virttest/qemu_devices/qdevices.py

```diff
@@ -927,8 +927,22 @@ def verify_hotplug(self, out, monitor):
         return False
 
 
-class Dimm(QDevice):
+class Nvdimm(QDevice):
```

Comment:
Nvidmm hotplug/unplug is not supported yet. I reused the dimm class, and will update the hotplug function when it is supported. Please help review again. Thanks!